dogs/components/dogs.js

```diff
@@ -23,9 +22,9 @@ class Dogs extends React.Component {
       <span>MY DOGS</span>
       <div className={styles.dogsContainer}>
         {
-          map(dogs, (dog, i) => {
+          values(map((dog, i) => {
```

Comment:
can you move this up to a higher scope?

in general, the value of currying functions like ramda is that you can _declare_ the functions close to the top level, since the data argument is last, then use them in your functions with data. see also [`pipe`](http://ramdajs.com/docs/#pipe) when you need to do a chain of operations.